lib/core/uri_parser.js

```diff
@@ -416,6 +415,24 @@ function parseQueryString(query, options) {
     console.warn('Unsupported option `wtimeout` specified');
   }
 
+  if (
+    result.w != null &&
+    result.w === 0 &&
```

Comment:
`x === 0` or `x === true` implies `x != null`. This can be written as:

```js
if (result.w === 0 && (result.j === true || result.journal === true))
```